client/component/src/PlotToolbar.tsx

```diff
@@ -108,13 +108,13 @@ function PlotToolbar(props: PropsWithChildren): JSX.Element {
           console.log('Setting current selection', last.id);
           setCurrentSelectionID(last.id);
           if (showSelectionConfig) {
-            enableSelection(last);
+            dashSelection(last);
```

Comment:
Can we also just set asDashed = false in the function on L104?

Reply:
Instead of `dashSelection` function, can call `s.asDashed = true;` directly and on L104 call `s.asDashed = false;` directly without `s.fixed = false;`

Reply:
You could rename and change `disableSelection`

Reply:
Changed to `dashSelection`